tests/test_laser_envelope.py

```diff
@@ -45,9 +45,9 @@
 use_cuda = True
 
 # Simulation box
-Nz = 50
-zmin = -10.e-6
-zmax = 10.e-6
+Nz = 200
+zmin = -20.e-6
+zmax = 20.e-6
```

Comment:
Could you keep the previous `Nz` and `zmin`/`zmax`, and then reduce the time step for the test with the periodic box (basically use the same time step as for the moving window)?
This would help use remember that there is a problem for large timesteps...